addons/web_editor/static/src/js/editor/odoo-editor/test/spec/powerbox.test.js

```diff
@@ -196,6 +196,7 @@ describe('Powerbox', () => {
         });
         it('should filter commands with filter text', async () => {
             const editable = document.createElement('div');
+            editable.classList.add('odoo-editor-editable');
```

Comment:
Why do we suddenly need this?

Reply:
When test case [`setSelection`](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/editor/odoo-editor/test/spec/powerbox.test.js#L203) on editable.firstChild from `setSelection` it moves to [`getNormalizedCursorPosition`](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js#L481) wherein it does not find editable so after setSelection the selection.anchorNode would be entire document. After my fix [`powerbox._context.initialValue`](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/editor/odoo-editor/test/spec/powerbox.test.js#L217) would compute wrong [`initialValue`](https://github.com/odoo/odoo/pull/127706/commits/431128d60486fd55354036e355bd4bac6196e3f8#diff-5771e6eec0f282eb6f2e53562ad34c786e9a6fcff10ffcc3db40303152cf46c2R121) because `selection.anchorNode.textContent` would be the textContent of whole document.